src/features/session_replay/instrument/index.component-test.js

```diff
@@ -0,0 +1,169 @@
+import { Instrument as SRInstrument } from './index'
+import { Aggregator } from '../../../common/aggregate/aggregator'
+import { getConfigurationValue, originals } from '../../../common/config/config'
+import { PREFIX, DEFAULT_KEY, MODE } from '../../../common/session/constants'
+
+jest.mock('../../../common/constants/runtime', () => ({
+  __esModule: true,
+  isBrowserScope: true,
+  globalScope: global
+}))
+jest.mock('../../../common/window/load', () => ({
+  __esModule: true,
+  onWindowLoad: jest.fn(cb => cb())
+}))
+jest.mock('../../../common/config/config', () => ({
+  __esModule: true,
+  originals: {
+    MO: true
+  },
+  getConfigurationValue: jest.fn()
+}))
+jest.mock('../aggregate', () => ({
+  __esModule: true,
+  Aggregate: jest.fn()
+}))
+jest.mock('../../utils/agent-session', () => ({
+  __esModule: true,
+  setupAgentSession: jest.fn().mockReturnValue({
+    isNew: true
+  })
+}))
+jest.mock('../shared/recorder', () => ({
+  __esModule: true,
+  Recorder: jest.fn().mockImplementation(() => {
+    return {
+      startRecording: jest.fn(),
+      stopRecording: jest.fn()
+    }
+  })
+}))
```

Comment:
Curious if we should try to dedicate some time to better the testing experience here so we are not having to do so many mocks.

Reply:
i'm good either way